components/storage/utils.ts

```diff
@@ -1,110 +1,158 @@
-import { ServiceConfig } from "@/lib/storage-types"
+import {FormQuestions, SelectedAnswers, ServiceConfig,} from "@/lib/storage-types"
 
 /**
- * Extracts all unique feature names from an array of services.
- * @param services Array of ServiceConfig objects
- * @returns Array of unique feature names (strings)
+ * Determines whether text should be white or black based on the background color
```

Comment:
i think this function is now in the Badge component because it is reused in featured carousel for now, and maybe other places if badges are used elsewhere. maybe just import from Badge?